node-tests/integration/commands/make-test.js

```diff
@@ -157,4 +162,30 @@ describe('electron:make command', function () {
     rimraf.sync('electron-app');
     await expect(command.validateAndRun([])).to.be.rejected;
   });
+
+  it('can publish', async function () {
```

Comment:
This is a great start on the tests. It's currently only testing the command code and not the task code, though, because the task is being stubbed rather than the forge API method. So let's stop stubbing the task and instead use the forge API stub to verify results. Then we can also verify that the `dir` argument is being passed correctly, and that the make results are being conveyed properly.

So, this test would look like:

```js
let makeResults = {};

// Change the make stub to return some fake results so we can ensure they
// they are passed to the publish API method
api.make.resetBehavior();
api.make.resolves(makeResults);

await expect(command.validateAndRun(['---publish'])).to.be.fulfilled;
expect(api.publish).to.be.calledOnce;
expect(api.publish.firstCall.args[0].dir).to.equal('electron-app');
expect(api.publish.firstCall.args[0].makeResults).to.equal(makeResults);
expect(api.publish.firstCall.args[0].publishTargets).to.be.undefined;
```

Then in the next two tests you can skip the make results stubbing since we've already verified that, and just assert that `api.publish` was called and that the `publishTargets` are correct.